### DISCUSSION TITLE
Color inside the app do not match the widget color

### DISCUSSION THREAD

**Discussion by ericviana** [iSoron/uhabits#1518] [Help & FAQ]
The colors on the widget are much more saturated compared to inside the app. Is there a reason for that?

![b4dfe701-e6bf-46f1-9a41-0908c684c70c](https://user-images.githubusercontent.com/76598237/192587314-d7a36235-ea83-4339-bdba-2f0b0d7c2137.jpeg)
![51c3549a-0114-4f71-8c0d-0b5f2cf263e6](https://user-images.githubusercontent.com/76598237/192587325-c489008d-c8fe-41a3-80d4-1f7604c1bd46.jpeg)

**Comment by KosmicGR** [comment]
This is not the only problem that widgets have with colors.
![Screenshot 2023-02-06 173815](https://user-images.githubusercontent.com/40884601/217016050-cee9b8b6-ef68-433c-9641-1011854f4bb2.png)
This happens because they use light mode's color scheme. It makes weekly habits pretty much unreadable in widget form.

Here it is within the app
**Light Mode**:
![Screenshot 2023-02-06 173850](https://user-images.githubusercontent.com/40884601/217016068-24110d5d-2d8d-4f6a-91a3-bb0c9d2e80ee.png)
**Dark Mode**:
![Screenshot 2023-02-06 173921](https://user-images.githubusercontent.com/40884601/217016079-9e836b13-0269-4635-9a99-d8f4a0eca8a0.png)

For context, this is the color I picked as an example. Any of the last 3 colors at the bottom row have this problem.
![Screenshot 2023-02-06 173940](https://user-images.githubusercontent.com/40884601/217016092-f1405aad-fa77-4db2-a034-b696ec504161.png)

Hopefully, there will be a fix of this coming soon, because without it, organization of habits becomes limited _if you use widgets_ because these 3 colors are broken.

**Comment by DataJoy-VJK** [comment]
I use Android 14 on a Pixel 7 and most, but not all, of my widgets are just black no matter what color they are in the app.
